pkgs/development/interpreters/python/pypy/default.nix

```diff
@@ -151,7 +151,7 @@ in with passthru; stdenv.mkDerivation rec {
     homepage = "http://pypy.org/";
     description = "Fast, compliant alternative implementation of the Python language (${pythonVersion})";
     license = licenses.mit;
-    platforms = [ "i686-linux" "x86_64-linux" "x86_64-darwin" ];
+    platforms = [ "aarch64-linux" "i686-linux" "x86_64-linux" "x86_64-darwin" ];
```

Comment:
Shouldn't we just use `platforms = platforms.unix;` at this point?

Reply:
Maybe? Does it work on `ppc`?

Reply:
A quick google leads me to believe "probably no"

Reply:
My point being that pypy _doesn't_ just support posix OS', but relies heavily on the underlying ISA

Reply:
Sounds right. Thanks for explanation.